### DISCUSSION TITLE
2023년 03월 24일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#20] [General]
# 2023년 03월 24일

LOG 말랑과 말을 놓았다. 말랑이 실수해서 말을 놨는데 ㅋㅋㅋ 나는 말 놓는 거 좋아서 놓자고 했다. 저녁에 3단계 구현하는 거 말랑이 도와줬다. 말랑도 4단계 전까지는 상태 패턴 썼다고 해서 도움을 많이 받았다. THANK YOU, MALLANG…

https://cyma-s.github.io/2023-03-24/

**Comment by ezzanzzan** [comment]
화이팅 ♥♥♥
